win32ss/user/ntuser/windc.c

```diff
@@ -488,9 +680,9 @@ UserGetDCEx(PWND Wnd OPTIONAL, HANDLE ClipRegion, ULONG Flags)
          Dce = DceAllocDCE(NULL, DCE_CACHE_DC);
       }
       if (Dce == NULL) return NULL;
-
-      Dce->hwndCurrent = (Wnd ? UserHMGetHandle(Wnd) : NULL);
-      Dce->pwndOrg = Dce->pwndClip = Wnd;
+      
```

Comment:
Strip EOL whitespace. Best is to use the NPP feature, in case this is not the only one.
```suggestion

```